R/animate_tour.R

```diff
@@ -84,6 +84,12 @@ animate_tour <- function(data,
     bases,
     render_opts$aps / render_opts$fps
   ))
+
+  basis_indices <- numeric(0)
+  if (!rlang::is_null(attr(projection_matrices, "new_basis"))) {
+    basis_indices <- which(attr(projection_matrices, "new_basis")) - 1
+  }
+
```

Comment:
Why do you initiate `numeric(0)`? Is it equivalent to `NULL` here? If so, below is the shortcut.

```suggestion
basis_indices <- NULL %||% which(attr(projection_matrices, "new_basis")) - 1
```

Reply:
The `numeric(0)` behaves differently when it's converted to json. I.e. `numeric(0)` becomes `[]` whereas `NULL` becomes `null`

Reply:
Gotta.